operator/src/test/java/oracle/kubernetes/operator/WatcherTestBase.java

```diff
@@ -101,10 +105,20 @@ void sendInitialRequest(BigInteger initialResourceVersion) {
     createAndRunWatcher(NAMESPACE, stopping, initialResourceVersion);
   }
 
+  DomainWatcher sendBookmarkRequest(BigInteger initialResourceVersion, String bookmarkResourceVersion) {
```

Comment:
Generally, we've kept the type of watcher generic in the test base. This could simply return Watcher<?> and the unit test could either cast the result or just accept the generic.